src/tasks/setup/package.ts

```diff
@@ -12,6 +12,7 @@ export default function taskClean(settings: IGulpSettings): any {
 
     let dependencies: any;
     const devDependencies: any = {
+        "gulp": `${settings.shenanigans.dependencies.gulp}`,
```

Comment:
This will require a little more typing support so it doesn't become confusing if a new project forgets to include gulp. Can you change the type for `dependencies` to also include a `gulp` key?